### PR TITLE
Operator sts

### PR DESCRIPTION
**Pull request by pjuarezd**

### CODE REVIEW

File: resources/base/crds/minio.min.io_tenants.yaml [hunk 1]
```diff
@@ -1,5 +1,89 @@
 apiVersion: apiextensions.k8s.io/v1
 kind: CustomResourceDefinition
+metadata:
+  annotations:
+    controller-gen.kubebuilder.io/version: v0.4.7
+  creationTimestamp: null
+  name: policybindings.minio.min.io
```
Comment (dvaldivia) [inline resources/base/crds/minio.min.io_tenants.yaml:7]: this is duplicated on the `minio.min.io_policybindings.yaml` file

**Review by dvaldivia — CHANGES_REQUESTED** [review]

File: pkg/controller/cluster/main-controller.go [hunk 2]
```diff
@@ -203,7 +203,7 @@ type Controller struct {
 }
 
 // NewController returns a new sample controller
-func NewController(podName string, namespacesToWatch set.StringSet, kubeClientSet kubernetes.Interface, minioClientSet clientset.Interface, promClient promclientset.Interface, statefulSetInformer appsinformers.StatefulSetInformer, deploymentInformer appsinformers.DeploymentInformer, podInformer coreinformers.PodInformer, tenantInformer informers.TenantInformer, serviceInformer coreinformers.ServiceInformer, hostsTemplate, operatorVersion string) *Controller {
+func NewController(podName string, namespacesToWatch set.StringSet, kubeClientSet kubernetes.Interface, minioClientSet clientset.Interface, promClient promclientset.Interface, statefulSetInformer appsinformers.StatefulSetInformer, deploymentInformer appsinformers.DeploymentInformer, podInformer coreinformers.PodInformer, tenantInformer informers.TenantInformer, policyBindingInformer informers.PolicyBindingInformer, serviceInformer coreinformers.ServiceInformer, hostsTemplate, operatorVersion string) *Controller {
```
Comment (dvaldivia) [inline pkg/controller/cluster/main-controller.go:206]: `policyBindingInformer` is not used anywhere, we can remove it

**Review by dvaldivia — DISMISSED** [review]

File: pkg/controller/cluster/main-controller.go (hunk 2 above)
Comment (pjuarezd) [reply pkg/controller/cluster/main-controller.go]: removed for now, might add it later if needed

**Comment by pjuarezd** [comment]
rebased in [1414](https://github.com/minio/operator/pull/1414)  PR

**Pull request closed without merging**